src/App.js

```diff
@@ -1,23 +1,12 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Firebase from './pages/firebase';
```

Comment:
The p in pages in the import path is lowercase, but the directory file is uppercase.  When I started it up, it could not find the file unless I changed it.